packages/vscode-extension/src/debugging/RadonCDPProxyDelegate.ts

```diff
@@ -246,7 +255,43 @@ export class RadonCDPProxyDelegate implements CDPProxyDelegate {
     throw new Error("Source map URL schemas other than `data` and `http` are not supported");
   }
 
-  private async handleScriptParsed(command: IProtocolCommand): Promise<IProtocolCommand> {
+  private async setupRadonConnectRuntime(tunnel: ProxyTunnel) {
+    // load script from lib/connect_runtime.js and evaluate it
+    const runtimeScriptPath = path.join(
+      extensionContext.extensionPath,
+      "dist",
+      "connect_runtime.js"
+    );
+    const runtimeScript = fs.readFileSync(runtimeScriptPath, "utf8");
+
+    await tunnel.injectDebuggerCommand({
+      method: "Runtime.addBinding",
+      params: {
+        name: "__radon_binding",
+      },
+    });
+
+    const result = (await tunnel.injectDebuggerCommand({
+      method: "Runtime.evaluate",
+      params: {
+        expression: runtimeScript,
+      },
+    })) as Cdp.Runtime.EvaluateResult;
+    if (result.exceptionDetails) {
+      Logger.error("Failed to setup Radon Connect runtime", result.exceptionDetails);
```

Comment:
Maybe this should be handled by Radon? I'm not sure what the consequences are, but won't it cause the Preview to get stuck in some state?